airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +77,24 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        query = session.query(Variable).all()
```

Comment:
```suggestion
       rows = session.query(Variable).all()
```
Since this returns rows of data. :)